draft-ietf-doh-dns-over-https-latest.mkd

```diff
@@ -311,6 +312,16 @@ accept = application/dns-udpwireformat
 
 # The HTTP Response
 
+A successful DOH response will carry DNS information in the HTTP
+response with a status code between 200 and 299. A successful HTTP
+response indicates a valid DNS response to the query made in the
+HTTP request but that DNS level response itself might represent some
+sort of failure (e.g. NXDOMAIN). HTTP response codes that are not in
```

Comment:
The connection between "successful DOH response" and 2xx isn't concrete enough here.

An HTTP response with a 2xx status code indicates a valid DNS response to the query made in the HTTP request.  A valid DNS response includes both success and failure responses.